src/freertos_drivers/net_cc32xx/CC32xxWiFi.hxx

```diff
@@ -130,6 +141,11 @@ public:
                       SecurityType security_type);
 
 
+    /** Initiate a WPS Push Button Control connection.
```

Comment:
please document whether this blocks the current thread until the connection succeeds or not.
from the `@return` line it's not clear to me if it returns 0 when it successfully started or succesffully finished the WPS search.